kustomize/commands/edit/remove/removeconfigmap_test.go

```diff
@@ -92,16 +96,67 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 `,
 		},
+		"succeeds when one configmap name exists in the specified namespace": {
+			input: `
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+configMapGenerator:
+- name: test-cm
+  namespace: test-ns
```

Comment:
Should we expect a ConfigMap to remain when it is in a namespace other than default and the `--namespace` flag is omitted?

Could we add this scenario to the tests?